.zenodo.json

```diff
@@ -78,7 +78,97 @@
       "orcid": "0000-0001-8421-365X"
     },
     {
-     "affiliation": "Université Paris-Saclay, Inria, CEA, Palaiseau, France",
+      "affiliation": "",
+      "name": "Ariel Rokem ",
+      "orcid": "0000-0003-0679-1985"
+     },
+     {
+      "affiliation": "",
+      "name": "Austin Hurst",
+      "orcid": "0000-0002-0219-4131"
+     },
+     {
+      "affiliation": "",
+      "name": "Diego Lozano-Soldevilla",
+      "orcid": "0000-0003-1794-8204"
+     },
+     {
+      "affiliation": "",
+      "name": "Eduard Ort",
+      "orcid": "0000-0001-5546-3561"
+     },
+     {
+      "affiliation": "",
+      "name": "Ethan Knights",
+      "orcid": "0000-0001-6078-9160"
```

Comment:
@ethanknights please check and/or comment with missing details!

Reply:
Thanks, all correct &  my affiliation is:

` "affiliation": "Medical Research Council Cognition & Brain Sciences Unit, University of Cambridge",`